### PR TITLE
Document time units. Fixes #1277

### LINKED ISSUE #1277: [BUG] incorrect x-axis range(date) in stacked area plot
Simply, I am counting the number of items against time intervals.

Code:

```
{
  "data": {
    "url": "https://www.cs.helsinki.fi/u/hxiao/event_html/data/ukraine/vega/data.json"
  },
  "mark": "area",
  "encoding": {
    "x": {
      "timeUnit": "yearmonthdayhours",
      "field": "datetime",
      "type": "temporal",
      "scale": {
        "nice": "hour"
      },
      "axis": {
        "axisWidth": 0,
        "format": "%m-%d %H:00",
        "labelAngle": 15
      }
    },
    "y": {
      "aggregate": "count",
      "field": "*"
    },
    "color": {
      "field": "series",
      "type": "nominal",
      "scale": {
        "range": "category20"
      }
    }
  },
  "config": {
    "cell": {
      "width": 800,
      "height": 400
    }
  }
}
```

The bug is on range of x-axis:

- **Actual**: Feb 8 to Mar, 1.
- **Expected**: Feb-27 to Mar-03(min and max of the datetime in all items)

Or the web app: https://www.cs.helsinki.fi/u/hxiao/event_html/vega.html

### PR DESCRIPTION
**Pull request by domoritz**
cc @bobocandys

### CODE REVIEW

File: site/docs/timeunit.md [hunk 1]
```diff
@@ -25,10 +25,26 @@ permalink: /docs/timeunit.html
...
+
+Vega-Lite supports the following time units, which can be combined into the `timeUnit` properties above (e.g. `yearmonthday`, `hoursminutes`).
+
+| Function      | Description    |  Example time unit of `Monday April 4, 2016 11:52:34:0201`  |
+| :------------ | :------------- | :-----------------------------------------------|
+| `date`        | returns the day of the month for a given date input, in local time | `4` |
+| `day`   | return the day of the week for a given date input, in local time | `1` |
+| `year`        | returns the year for a given date input, in local time | `2016` |
+| `month`       | returns the (zero-based) month for a given date input, in local time | `3` |
+| `hours`       | returns the hours component for a given date input, in local time | `11` |
```
Comment (kanitw) [inline site/docs/timeunit.md]: 0-23?

File: site/docs/timeunit.md [hunk 2]
```diff
@@ -25,10 +25,26 @@ permalink: /docs/timeunit.html
...
+| `minutes`     | returns the minutes component for a given date input, in local time | `52` |
+| `seconds`     | returns the seconds component for a given date input, in local time | `34` |
+| `milliseconds`| returns the milliseconds component for a given date input, in local time | `0201` |
```
Comment (kanitw) [inline site/docs/timeunit.md]: `201`?

File: site/docs/timeunit.md [hunk 3]
```diff
@@ -25,10 +25,26 @@ permalink: /docs/timeunit.html
...
 
+
+## Date/Time Units
+
+Vega-Lite supports the following time units, which can be combined into the `timeUnit` properties above (e.g. `yearmonthday`, `hoursminutes`).
+
+| Function      | Description    |  Example time unit of `Monday April 4, 2016 11:52:34:0201`  |
+| :------------ | :------------- | :-----------------------------------------------|
+| `date`        | returns the day of the month for a given date input, in local time | `4` |
+| `day`   | return the day of the week for a given date input, in local time | `1` |
```
Comment (kanitw) [inline site/docs/timeunit.md]: 0 for Sunday, 1 for Monday, 2 for Tuesday, and so on.

File: site/docs/timeunit.md (hunk 1 above)
Comment (domoritz) [inline site/docs/timeunit.md]: What else?

Comment (kanitw) [inline site/docs/timeunit.md]: it could be 0-11 with a separate AM PM function.

**Pull request merged**